Source/WebCore/dom/ContentVisibilityDocumentState.cpp

```diff
@@ -117,6 +120,16 @@ DidUpdateAnyContentRelevancy ContentVisibilityDocumentState::updateRelevancyOfCo
             if (relevancyToCheck.contains(ContentRelevancy::Focused))
                 setRelevancyValue(ContentRelevancy::Focused, target->hasFocusWithin());
 
+            auto targetContainsSelection = [](Element& target) {
+                auto selectionRange = target.document().selection().selection().range();
+                if (!selectionRange)
+                    return false;
+                return intersects(*selectionRange, target);
```

Comment:
```suggestion
                return intersects<ComposedTree>(*selectionRange, target);
```

Can we use the flat tree to match CSS here? That's not the default.

Reply:
Also I realize this can just be `selectionRange && intersects<ComposedTree>(*selectionRange, target);`

Reply:
Yeah, that is nicer, will fix before landing.